Formula/dislocker.rb

```diff
@@ -0,0 +1,20 @@
+class Dislocker < Formula
+  desc "FUSE driver to read/write Windows' BitLocker-ed volumes"
+  homepage "https://github.com/Aorimn/dislocker"
+  url "https://github.com/Aorimn/dislocker/archive/v0.7.1.tar.gz"
+  sha256 "742fb5c1b3ff540368ced54c29eae8b488ae5a5fcaca092947e17c2d358a6762"
+
+  depends_on "cmake" => :build
+  depends_on "mbedtls"
+  depends_on :osxfuse
+
+  def install
+    system "cmake", "-DCMAKE_DISABLE_FIND_PACKAGE_Ruby=TRUE", *std_cmake_args
+    system "make"
+    system "make", "install"
+  end
+
+  test do
+    system "#{bin}/dislocker", "-h"
```

Comment:
Can _anything_ be tested here in terms of functionality? Possibly not, given it needs `fuse`, but. Ideally.

Reply:
No, I don't think because it need a bitlocker encrypted volume.